src/app/pages/admin/admin.controller.js

```diff
@@ -12,10 +12,13 @@ export default class AdminController {
         this.listOfUsers = this.rootScope.listOfUsers;
 
         this.scope.filteredItems = [];
```

Comment:
don't use scope, put all variable to `this`

Reply:
fixed it